src/EditorFeatures/Test2/Rename/RenameCommandHandlerTests.vb

```diff
@@ -18,8 +19,9 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
     <[UseExportProvider]>
     Public Class RenameCommandHandlerTests
         Private Function CreateCommandHandler(workspace As TestWorkspace) As RenameCommandHandler
-            Return New RenameCommandHandler(workspace.GetService(Of InlineRenameService),
-                                            workspace.GetService(Of IEditorOperationsFactoryService))
+            Return New RenameCommandHandler(
+                workspace.ExportProvider.GetExportedValue(Of IThreadingContext)(),
```

Comment:
Why mixing ExportProvider and GetService?

Reply:
➡️ `IThreadingContext` is exported directly (not a workspace service)